WordPress/Classes/System/WordPressAppDelegate.m

```diff
@@ -1087,7 +1088,7 @@ - (void)setupUserAgent
     UIWebView *webView = [[UIWebView alloc] init];
     NSString *defaultUA = [webView stringByEvaluatingJavaScriptFromString:@"navigator.userAgent"];
 
-    NSString *appVersion = [[[NSBundle mainBundle] infoDictionary] objectForKey:@"CFBundleVersion"];
+    NSString *appVersion = [[[NSBundle mainBundle] infoDictionary] objectForKey:@"CFBundleShortVersionString"];
```

Comment:
I didn't use the new helper for this portion as I thought it might make for an odd user agent string to include both the `CFBundleVersion` and the `CFBundleShortVersionString`. That being said given we're using these new build numbers it seemed right to update this to use `CFBundleShortVersionString`.